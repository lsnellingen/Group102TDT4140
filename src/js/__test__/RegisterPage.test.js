import React from 'react';
import ReactDOM from 'react-dom';
import RegisterPage from '../pages/RegisterPage';
import renderer from 'react-test-renderer';
import { RegistrationForm, LoginLink } from 'react-stormpath';
import {shallow} from 'enzyme';

describe('RegisterPage', () => {
  var ReactTestUtils = require('react-addons-test-utils') // ES5 with npm

    test('renders without crashing', () => {
      /*const renderer = ReactTestUtils.createRenderer();*/

      const tree = renderer.create(
        <RegisterPage/>).toJSON();


      expect(tree).toMatchSnapshot();




    });
});
