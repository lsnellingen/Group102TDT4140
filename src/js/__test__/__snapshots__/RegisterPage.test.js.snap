// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegisterPage renders without crashing 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-xs-12"
    >
      <h3>
        Registration
      </h3>
      <hr />
    </div>
  </div>
  <form
    onSubmit={[Function]}
  >
    <div>
      <div
        className="sp-login-form"
      >
        <span />
        <div
          className="row"
          data-spIf="!account.created"
        >
          <div
            className="col-xs-12"
          >
            <div
              className="form-horizontal"
            >
              <div
                className="form-group"
              >
                <label
                  className="col-xs-12 col-sm-4 control-label"
                  htmlFor="givenName"
                >
                  First Name
                </label>
                <div
                  className="col-xs-12 col-sm-4"
                >
                  <input
                    className="form-control"
                    defaultValue=""
                    id="givenName"
                    name="givenName"
                    onChange={[Function]}
                    placeholder="First Name"
                    required={true}
                    type="text"
                  />
                </div>
              </div>
              <div
                className="form-group"
              >
                <label
                  className="col-xs-12 col-sm-4 control-label"
                  htmlFor="surname"
                >
                  Last Name
                </label>
                <div
                  className="col-xs-12 col-sm-4"
                >
                  <input
                    className="form-control"
                    defaultValue=""
                    id="surname"
                    name="surname"
                    onChange={[Function]}
                    placeholder="Last Name"
                    required={true}
                    type="text"
                  />
                </div>
              </div>
              <div
                className="form-group"
              >
                <label
                  className="col-xs-12 col-sm-4 control-label"
                  htmlFor="email"
                >
                  Email
                </label>
                <div
                  className="col-xs-12 col-sm-4"
                >
                  <input
                    className="form-control"
                    defaultValue=""
                    id="email"
                    name="email"
                    onChange={[Function]}
                    placeholder="Email"
                    required={true}
                    type="text"
                  />
                </div>
              </div>
              <div
                className="form-group"
              >
                <label
                  className="col-xs-12 col-sm-4 control-label"
                  htmlFor="password"
                >
                  Password
                </label>
                <div
                  className="col-xs-12 col-sm-4"
                >
                  <input
                    className="form-control"
                    defaultValue=""
                    id="password"
                    name="password"
                    onChange={[Function]}
                    placeholder="Password"
                    required={true}
                    type="password"
                  />
                </div>
              </div>
              <div
                className="form-group"
              >
                <div
                  className="col-sm-offset-4 col-sm-4"
                >
                  <span />
                  <button
                    className="btn btn-info"
                    disabled={false}
                    type="submit"
                  >
                    Register
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </form>
</div>
`;
